.travis.yml

```diff
@@ -0,0 +1,19 @@
+# Travis CI script
+
+sudo: false
+
+services:
+- docker
+
+before_install:
+- docker pull fedora
+- docker run -d --rm --name AUTHSELECT-FEDORA -v `pwd`:/buildroot fedora /bin/sh -c "tail -f /dev/null"
+- docker ps
+- docker exec AUTHSELECT-FEDORA dnf -y install gcc make gettext gettext-devel autoconf automake libtool m4 pkgconfig popt-devel asciidoc libcmocka-devel libselinux-devel file
```

Comment:
Is the `file` at the end of the line a typo or is it actually doing something?

Reply:
It is not typo, file utility is used somehow in configure. Configure complained when running (some pipe check) that file utility is not present.